provisioning/service/src/Auth/ProvisioningTokenCredential.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System.Threading;
+using Azure.Core;
+using Microsoft.Azure.Devices.Common.Service.Auth;
+
+namespace Microsoft.Azure.Devices.Provisioning.Service.Auth
+{
+    /// <summary>
+    /// Allows authentication to the API using a JWT token generated for Azure active directory.
+    /// The PnP client is auto generated from swagger and needs to implement a specific class to pass to the protocol layer
+    /// unlike the rest of the clients which are hand-written. so, this implementation for authentication is specific to digital twin (Pnp).
+    /// </summary>
+    internal class ProvisioningTokenCredential : IAuthorizationHeaderProvider
+    {
+        private const string _tokenType = "Bearer";
```

Comment:
I fixed this in the code you copied from, but it should be `TokenType` since it is a constant.

Reply:
But now that you don't have 2 instances of it, you could move the constant to be inline.